lib/controllers/createController.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var path = require('path');
+
+module.exports = function (apiPlatformService, fileSystemRepository,
+    workspaceRepository) {
+  return {
+    create: create
+  };
+
+  function create(newApi) {
+    return apiPlatformService.createAPI(newApi.bizGroup, newApi.api,
+        newApi.apiVersion, newApi.rootRamlPath)
+      .then(function (api) {
+        return workspaceRepository.get()
+          .then(function (workspace) {
```

Comment:
avoid nesting then calls this way. Store needed values in function variables or use bluebird props